static/payment.css

```diff
@@ -0,0 +1,30 @@
+ /* * {font-family: "Helvetica Neue", Helvetica; font-size: 15px; font-variant: normal; padding: 0; margin: 0;} */
+
+ /* html {height: 100%;} */
+
+ /* body { */
+    /* background: #E6EBF1; */
+    /* align-items: center; */
+    /* min-height: 100%; */
+    /* display: flex; */
+    /* width: 100%; */
+ /* } */
+
+form {width: 480px; margin: 20px auto;} 
```

Comment:
Think of this as WIP. You know how it looks when I "design" things...

This is mostly just copied from Stripe's examples, with some slight modifications on my part. The "minification" was done by me hitting J a bunch of times in my editor to easily be able to prefix a bunch of the rules with `.payment-form`, or something like that.

I'll tidy this up eventually. I'm also thinking to pull in webpack to package this up properly, but not in this PR.